homeassistant/components/homematicip_cloud/config_flow.py

```diff
@@ -54,6 +54,8 @@ def __init__(self):
                 vol.Required(HMIPC_HAPID): str,
                 vol.Optional(HMIPC_NAME): str,
                 vol.Optional(HMIPC_PIN): str,
+                vol.Required(HMIPCS_ENABLE_GROUP_SWITCHES): bool,
```

Comment:
We don't want these kind of options in the config flow. The config flow is mainly for connecting and authentication. When we have an options flow for config entries, it can be implemented with that. For now, please remove the options from the config flow and just keep them in config yaml.

Reply:
Ok, options are removed now.